test/e2e/network/dns.go

```diff
@@ -53,16 +54,37 @@ var _ = SIGDescribe("DNS", func() {
 			namesToResolve = append(namesToResolve, "google.com")
 			namesToResolve = append(namesToResolve, "metadata")
 		}
-		wheezyProbeCmd, wheezyFileNames := createProbeCommand(namesToResolve, nil, "", "wheezy", f.Namespace.Name, framework.TestContext.ClusterDNSDomain)
-		jessieProbeCmd, jessieFileNames := createProbeCommand(namesToResolve, nil, "", "jessie", f.Namespace.Name, framework.TestContext.ClusterDNSDomain)
+		wheezyProbeCmd, wheezyFileNames := createProbeCommand(
+			isIPv6,
+			namesToResolve,
+			nil,
+			"",
+			"wheezy",
+			f.Namespace.Name,
+			framework.TestContext.ClusterDNSDomain)
+		jessieProbeCmd, jessieFileNames := createProbeCommand(
+			isIPv6,
+			namesToResolve,
+			nil,
+			"",
+			"jessie",
+			f.Namespace.Name,
+			framework.TestContext.ClusterDNSDomain)
 		By("Running these commands on wheezy: " + wheezyProbeCmd + "\n")
 		By("Running these commands on jessie: " + jessieProbeCmd + "\n")
 
 		// Run a pod which probes DNS and exposes the results by HTTP.
 		By("creating a pod to probe DNS")
 		pod := createDNSPod(f.Namespace.Name, wheezyProbeCmd, jessieProbeCmd, dnsTestPodHostName, dnsTestServiceName)
 		validateDNSResults(f, pod, append(wheezyFileNames, jessieFileNames...))
-	})
+	}
+
+	// only IPv4 is conformance for now
```

Comment:
nit: let's list this as a TODO:

Reply:
Are you sure.
the TODO would be to run IPv6 Conformance tests .. it is a k8s-wide tests issue. Not only this one.

Reply:
I removed that comment.